packages/block-library/src/search/edit.js

```diff
@@ -168,6 +218,60 @@ export default function SearchEdit( {
 		);
 	};
 
+	const renderInputs = () => {
+		const wrapperStyles =
+			'button-inside' === buttonPosition
+				? {
+						borderRadius: borderRadius
+							? borderRadius + 'px'
+							: undefined,
+				  }
+				: {};
+
+		const wrapperClasses = classnames(
+			'wp-block-search__inside-wrapper',
+			'button-inside' === buttonPosition ? sharedClasses : undefined
+		);
+
+		return (
+			<ResizableBox
+				size={ {
+					width: `${ width }${ widthUnit }`,
+				} }
+				className={ wrapperClasses }
+				style={ wrapperStyles }
+				isResetValueOnUnitChange
```

Comment:
This prop throws a console warning - `react-dom.82e849f1.js:82 Warning: React does not recognize the `isResetValueOnUnitChange` prop on a DOM element`.

It seems like this prop belongs to `UnitControl` component rather that `ResizableBox` ?

Reply:
I've removed this as I couldn't see where it would be used. After removing it all the resize functionality still works for me and no warning in console. Also, tweaked the SVGs' stroke width attributes to prevent similar error.